packages/wallet/api/test/test-lib-wallet.js

```diff
@@ -1358,13 +1358,20 @@ test('addOffer makeContinuingInvitation', async t => {
   await wallet.addOffer(offer);
 
   const uiNotifierP = wallet.getUINotifier(rawId, `unknown`);
+  const publicNotifiersP = wallet.getPublicNotifiers(rawId, `unknown`);
 
   const accepted = await wallet.acceptOffer(id);
   assert(accepted);
 
   const update = await E(uiNotifierP).getUpdateSince();
   t.is(update.value, 'first offer made');
 
+  // Test getNotifiers as well, even though it's redundant in this example.
+  const publicNotifiersUpdate = await E(
+    E.get(publicNotifiersP).notifier,
+  ).getUpdateSince();
+  t.is(publicNotifiersUpdate.value, 'first offer made');
```

Comment:
WDYT of testing that they're the same?
```suggestion
  t.deepEqual(publicNotifiersUpdate, update);
```

Reply:
I prefer this way because if/when we remove the other notifier, they're not as intertwined